proxies/live/apiproxy/resources/jsc/SelectIdTokenIssuer.js

```diff
@@ -0,0 +1,17 @@
+var issuer = context.getVariable('jwt.DecodeJWT.FromSubjectTokenFormParam.decoded.claim.iss');
+var idTokenIssuer = ''
+
+
+if (issuer.includes('NHSIdentity')) {
+    idTokenIssuer = 'nhsCis2'; 
+    jwksPath = context.getVariable('identity-service-config.cis2.jwks_path');
+} else if (issuer.includes('signin.nhs.uk') || issuer.includes('login.nhs.uk')) {
+    idTokenIssuer = 'nhsLogin';
+    jwksPath = context.getVariable('identity-service-config.nhs_login.jwks_path');
+} else {
+    print('error');
```

Comment:
Rather than printing the error would it be better to create an error variable and raise a fault?